interpreter.js

```diff
@@ -3593,7 +3593,7 @@ Interpreter.prototype['stepIdentifier'] = function(stack, state, node) {
   if (this.getterStep_) {
     // Call the getter function.
     var scope = state.scope;
-    while (!this.hasProperty(scope, node['name'])) {
+    while (scope !== this.globalScope && !this.hasProperty(scope, node['name'])) {
       scope = scope.parentScope;
     }
```

Comment:
@NeilFraser: what is the purpose of this code?  `scope` does not appear to be used.

Reply:
Indeed, this code served no purpose.  Removed here, with some historical explanation: af7b7c269db40a55787e15d8219cd58ce1b6ca73